### DISCUSSION TITLE
Welcome to BierBot-Brick Discussions!

### DISCUSSION THREAD

**Discussion by BernhardSchlegel** [BernhardSchlegel/BierBot-Bricks#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jonathanschneider** [comment]
Whoa, I only stumbled over this project because of the issue in BierBot! What an awesome idea! I immediately ordered a TH16.
I had been working on a feature for BierBot to control the kettle and the stirrer through Wi-Fi sockets via HTTP but I guess I’m too late to the party...

**Reply by BernhardSchlegel** [reply]
Hi Jonathan ! Welcome aboard ! Glad you like it :) Although not my main focus for now, there are still people preferring the BierBot (non Brick) because of it's indepence from the outside world. I strongly want to encourage you to finish your HTTP socket feature  - that sounds amazing!

**Reply by jonathanschneider** [reply]
Are you planning on supporting more devices besides the TH16 since a „brick“ for the stirrer wouldn’t necessarily need a thermometer?

**Reply by BernhardSchlegel** [reply]
The TH16 is unbeatable in terms of price. You can spare the DS18B20 temperature sensor and order your TH16 in china (if you have enough time to wait for the delivery) so you're looking at 10USD. I don't think you'll find a cheaper way to control your stirrer. 

**BUT** (for other reasons) I indeed plan to support other devices. A RaspberryPi, i.e. is appealing IHMO since it has plenty of GPIOs and multiple PWMs. One "Brick" would be capable to control most parts of your brewery (however, requires way more manual soldering and wiring).

I envision a simple Python/NodeJS app, that can be launched on a RaspberryPi with minimal depedencies that server a WebUI locally for (de)-activating GPIOs/PWMs and exposes them to the Bricks backend. I'm currently documenting the API for the backend. As soon as that is done, the "way is clear" for an experimental RasberryPi Brick software! Contributors wanted! ;)

Let me know what you think!

**Reply by jonathanschneider** [reply]
I was thinking of the Sonoff Basic. That’s only $5. A TH16 for the kettle and a Basic for the stirrer would make a pretty good team.
Or if one does not want to cut any cables, the S26.

**Reply by BernhardSchlegel** [reply]
There is a fair chance the existing firmware works just fine with those two options. 

Especially the **S26** is appealing since you don't have to worry about installing cables. 

The Sonoff Basic is only 6€ cheaper on Amazon. If you consider the cable (5€) the price difference is too small IMHO!

**Reply by jonathanschneider** [reply]
My first BierBot brick:

![IMG_0703](https://user-images.githubusercontent.com/4623773/120377249-2b214280-c31d-11eb-9792-8cb549520698.JPG)

Couldn't wait for the TH16 to arrive...

**Reply by BernhardSchlegel** [reply]
Love your drive!
